core/modules/node/tests/node.test

```diff
@@ -884,9 +911,76 @@ class NodeLayoutPreviewTestCase extends BackdropWebTestCase {
     // layout.
     $this->assertText('Node body field', '"Node body field" found.');
 
+    // Look for the heading of the custom block that exists only in our
+    // layout.
+    $this->assertText('Custom Node layout title', '"Custom Node layout title" found.');
+
     // Look for the body text, which appears only in that field if the context
     // was set properly.
-    $this->assertText($body, "\"{$body}\" found.");
+    $this->assertText($body, "Body text \"{$body}\" found.");
+
+    // Now enable the 'custom_node_preview' layout; give it path
+    // node/preview/{$this->content_type->type}/%, and run the same tests.
+
+    $layouts = layout_load_all();
+    $preview_layout = &$layouts['custom_node_preview'];
```

Comment:
The `&` is not really needed, as the value stored at `$layouts['custom_node_preview']` is a PHP object.

Reply:
Agreed.